crates/sncast/tests/e2e/call.rs

```diff
@@ -85,7 +85,7 @@ fn test_wrong_function_name() {
 
     snapbox.assert().stderr_matches(indoc! {r"
         command: call
-        error: Entry point [..] not found in contract.
+        error: Contract error
```

Comment:
I assume this reflects changes from the rpc update, right? Those old errors were nice. `Contract error` is not meaningful at all.

Reply:
that's true, I liked them better as well. I have created an issue (for now on our part) to talk to devnet guys and see if something's could be done with this.

Reply:
`Contract error` should also have `data` with `revert_error` that has more specific details, maybe it's just not currently handled?